src/bosh-director/spec/functional/gcs_spec.rb

```diff
@@ -79,5 +79,66 @@ module Bosh::Blobstore
       end
     end
 
+    context 'Read-Only', general_gcs: true do
+        let(:gcs_options) do
+          {
+            bucket_name: bucket_name,
+            credentials_source: 'none',
+            gcscli_path: gcscli_path
+          }
+        end
+
+      let(:gcs) do
+        Client.create('gcscli', gcs_options)
+      end
+
+      let(:contents) do
+        'foobar'
+      end
+
+      describe 'get object' do
+        it 'should save to a file' do
+          file = Tempfile.new('contents')
+          gcs.get('public', file)
+          file.rewind
+          expect(file.read).to include contents
```

Comment:
Switched to equality.

I had swapped it to include as part of debugging with a manually uploaded file.